ftplugin/php.vim

```diff
@@ -1,57 +1,16 @@
-if exists('g:phpactorLoaded')
-  finish
-endif
-
-let g:phpactorLoaded = 1
-let g:phpactorpath = expand('<sfile>:p:h') . '/..'
-let g:phpactorbinpath = g:phpactorpath. '/bin/phpactor'
-let g:phpactorInitialCwd = getcwd()
-let g:phpactorCompleteLabelTruncateLength=50
-
-""
-" Path to the PHP binary used by Phpactor
-let g:phpactorPhpBin = get(g:, 'phpactorPhpBin', 'php')
-
-""
-" The Phpactor branch to use when calling @command(PhpactorUpdate)
-let g:phpactorBranch = get(g:, 'phpactorBranch', 'master')
-
-""
-" Automatically import classes when using VIM native omni-completion
-let g:phpactorOmniAutoClassImport = get(g:, 'phpactorOmniAutoClassImport', v:true)
-
-""
-" Ignore case when suggestion completion results
-let g:phpactorCompletionIgnoreCase = get(g:, 'phpactorCompletionIgnoreCase', 1)
-
-""
-" Function to use when populating a list of code references. The default
-" is to use the VIM quick-fix list.
-let g:phpactorQuickfixStrategy = get(g:, 'phpactorQuickfixStrategy', 'phpactor#quickfix#vim')
-
-""
-" Function to use when presenting a user with a choice of options. The default
-" is to use the VIM inputlist.
-let g:phpactorInputListStrategy = get(g:, 'phpactorInputListStrategy', 'phpactor#input#list#inputlist')
-
-""
-" When jumping to a file location: if the target file open in a window, switch
-" to that window instead of switching buffers.  The default is false.
-let g:phpactorUseOpenWindows = get(g:, 'phpactorUseOpenWindows', v:false)
-
-""
-" The list of files that determine workspace root directory
-" if contained within
-let g:phpactorProjectRootPatterns = get(g:, 'phpactorProjectRootPatterns', ['composer.json', '.git', '.phpactor.json', '.phpactor.yml'])
-
-""
-" The list of directories that should not be considered as workspace root directory
-" (in addition to '/' which is always considered)
-let g:phpactorGlobalRootPatterns = get(g:, 'phpactorGlobalRootPatterns', ['/', '/home'])
-
-if g:phpactorOmniAutoClassImport == v:true
-    autocmd CompleteDone *.php call phpactor#_completeImportClass(v:completed_item)
-endif
+function! s:import_on_complete_done(completed_item) abort
+    if get(b:, 'phpactorOmniAutoClassImport', g:phpactorOmniAutoClassImport) != v:true
+      return
+    endif
+    echom string(get(b:, 'phpactorOmniAutoClassImport', g:phpactorOmniAutoClassImport))
```

Comment:
Is this a debug statement?

Reply:
it's possible, I don't remember using it but I will remove because it has nothing to do here ;)

Reply:
Removed.

By the way, do you think we should add the condition in `phpactor#_completeImportClass` instead of creating a temporary function as I did ?
The goal was to not change the way `phpactor#_completeImportClass` works in case someone wanted to use it but I'm not sure it really make sense.